pkg/util/flag/flags_test.go

```diff
@@ -207,7 +207,7 @@ func TestReservedMemoryVar(t *testing.T) {
 		},
 		{
 			desc: "valid input with multiple reserved-memory arguments",
-			argc: "blah --reserved-memory=0:memory=1Gi,hugepages-1Gi=1Gi --reserved-memory=1:memory=1Gi",
```

Comment:
Can we have both tests?
One for multiple arguments and the second for argument with multiple reservations divided via `;`